crates/libs/core/src/imp/mod.rs

```diff
@@ -1,21 +1,16 @@
-mod bindings;
+#[cfg(windows)]
+include!("windows.rs");
```

Comment:
I admit I don't love things like this, Which I guess illustrates the point on the impact of supporting non-Windows platforms.

Reply:
Exactly.